reference/storefront_tokens.v3.yml

```diff
@@ -69,6 +69,7 @@ paths:
                 -  'https://www.yourstorefront.com/'
               channel_id: 1
               expires_at: 1885635176
+              dev_mode: false
```

Comment:
perfectly `dev_mode: false` shouldn't be used - only when you need to enable it then you need to use `dev_mode: true`.

Reply:
fixed now 👍